src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentStatusRouteSpec.scala

```diff
@@ -254,4 +254,91 @@ class ConsignmentStatusRouteSpec extends TestContainerUtils with Matchers with T
       response.errors.head.message should equal(expectedResponse.errors.head.message)
       response.errors.head.extensions should equal(expectedResponse.errors.head.extensions)
   }
+
+  "updateConsignmentStatus" should "return an error if statusValue is missing and the statusType isn't 'Upload'" in withContainers {
+    case container: PostgreSQLContainer =>
+      val utils = TestUtils(container.database)
+      val userId = UUID.fromString("49762121-4425-4dc4-9194-98f72e04d52e")
+      val token = validUserToken(userId)
+      val consignmentId = UUID.fromString("a8dc972d-58f9-4733-8bb2-4254b89a35f2")
+      utils.createConsignment(consignmentId, userId)
+
+      val expectedResponse = expectedUpdateConsignmentStatusMutationResponse("data_missing_statusvalue_not_allowed")
+      val response = runUpdateConsignmentStatusTestMutation("mutation_missing_statusvalue_not_allowed", token)
+
+      response.errors.head.message should equal(expectedResponse.errors.head.message)
+      response.errors.head.extensions should equal(expectedResponse.errors.head.extensions)
+  }
+
+  "updateConsignmentStatus" should "return an error if statusType is 'Upload', statusValue wasn't passed in but no files have 'Upload' statuses" in withContainers {
```

Comment:
Maybe reword test title slightly to:` ... if statusType is 'Upload' and no statusValue is passed where consignment has no files with 'Upload' statuses`